R/nlme_methods.R

```diff
@@ -433,11 +436,11 @@ getVarCov.fh <- function(obj, individuals = 1, type = "random.effects", ...) {
         i <- as.character(obj$ind$Domain[obj$ind$Out == 0][i])
       }
       if (!(i %in% obj$ind$Domain[obj$ind$Out == 0])) {
-        stop(paste0(
-          "No variance-covariance matrix is available. Individual '",
-          i, "' is not contained in the sample and therefore not used
-                    for the model fitting."
-        ))
+        stop(strwrap(prefix = " ", initial = "",
+                     paste0("No variance-covariance matrix is available.
+                            Individual '", i, "' is not contained in the sample
+                            and therefore not used for the model fitting."
+                            )))
```

Comment:
```suggestion
        stop(strwrap(prefix = " ", initial = "",
                     sprintf("No variance-covariance matrix is available.
                            Individual '%d' is not contained in the sample
                            and therefore not used for the model fitting.", i
                            )))
```
Just personal opinion: for these kind of strings, `sprintf` is easier readable (and writable), but probably the package has its own style.